frontend/packages/core/src/Assets/colors.tsx

```diff
@@ -0,0 +1,195 @@
+// COLORS FROM FIGMA
+// NOTE THAT THOSE WITH GRADIENTS ONLY HAVE THEIR BASES
+
+// LIGHT MODE COLORS ////////////////////////////////////////////////////////
+
+// NEUTRAL
+export const CLUTCH_LIGHTMODE_NEUTRAL_5 = "#F8F8F9"; // PRIMARY BUTTON DISABLED / STROKE TERTIARY
```

Comment:
Thanks for adding this!

For the palette system I'm thinking it might work to have it defined a little differently.

Maybe
`const LIGHTMODE: CLUTCHPALETTE = {
NEUTRAL: {...}
};`

This will allow us to type everything so we can extend it easily later and then when we want to swap before palettes it's an easy reference change.

What do you think?

Reply:
good idea